calico/network-policy/encrypt-cluster-pod-traffic.mdx

```diff
@@ -34,22 +34,23 @@ This how-to guide uses the following {{prodname}} features:
 
 ## Before you begin...
 
-**Unsupported**
+**Terminology**
 
-- GKE
-- Using your own custom keys to encrypt traffic
+ - Inter-node pod traffic: Traffic leaving a pod from one node destined to a pod on another node
+ - Same-node pod traffic: Traffic between pods on the same node
```

Comment:
I would make same-node the last bullet so the items we support are listed first.